config_skeleton.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'diffy', '~> 3.0'
   s.add_runtime_dependency 'rb-inotify', '~> 0.9'
-  s.add_runtime_dependency 'service_skeleton', "~> 2.0"
+  s.add_runtime_dependency 'service_skeleton', '~> 2.0.1'
```

Comment:
I *think* this makes the dependency more specific... do we want that?

`~>2.0` means `>2 && <3`
`~>2.0.1` means `>2.0.1 && <2.1`

Given that service_skeleton *should* be following semver, I think we want the former?

Reply:
Yup, good point